### ISSUE TITLE
[YouTube] ERROR: Unable to extract yt initial data

### ISSUE THREAD

**Issue by Blandss** [ytdl-org/youtube-dl#32531]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x ] I've verified that I'm running youtube-dl version **2021.12.17**
- [ ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ ] I've searched the bugtracker for similar bug reports including closed ones
- [x ] I've read bugs section in FAQ

## Verbose log

```
nextcord.ext.commands.errors.CommandInvokeError: Command raised an exception: DownloadError: ERROR: Unable to extract yt initial data; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

WRITE DESCRIPTION HERE

My youtube-dl now working, I reinstalled it and my version is 2021.12.17

**Comment by dirkf** [comment]
I guess "...**not working**..."?

Review #30839 and #32499.

**Comment by TofPlay** [comment]
Same issue 
```
WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```
with version
```
yt-dlp -U
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
```
